includes/class-webmention-sender.php

```diff
@@ -63,22 +62,22 @@ public static function comment_post( $id ) {
 	}
 
 	/**
-	 * Send Webmention delete
+	 * Send Webmention delete.
 	 *
-	 * @param int $post_id
+	 * @param int $post_id Trashed post ID.
 	 */
 	public static function trash_hook( $post_id ) {
 		wp_schedule_single_event( time() + wp_rand( 0, 120 ), 'webmention_delete', array( $post_id ) );
 	}
 
 	/**
-	 * Send Webmentions
+	 * Send Webmentions.
 	 *
-	 * @param string $source source url
-	 * @param string $target target url
-	 * @param int $post_id the post_ID (optional)
+	 * @param string $source source url.
+	 * @param string $target target url.
+	 * @param int    $post_id the post_ID (optional).
 	 *
-	 * @return array of results including HTTP headers
+	 * @return array of results including HTTP headers.
 	 */
 	public static function send_webmention( $source, $target, $post_id = null ) {
 		// stop selfpings on the same URL
```

Comment:
Shouldn't this be also start uppercase?

Reply:
I am just asking, because I am not a native speaker and I am curious, what you would prefer.